src/components/pages/TopicCreation/TopicCreation.js

```diff
@@ -104,26 +79,29 @@ const TopicCreation = () => {
       ...currentTopic,
       [fieldName]: fieldValue,
     });
-    handleCurrentValidation();
   };
 
   //current step validation handler
   const handleCurrentValidation = () => {
     if (currentStep === 1) {
-      if (currentTopic.contextName.length !== 0) {
+      if (newContextType) {
         setStepValidation({ ...stepValidation, [currentStep]: true });
       }
+      // setStepValidation({ ...stepValidation, [currentStep]: true });
     } else if (currentStep === 2) {
-      if (currentTopic.name && currentTopic.frequency) {
+      if (currentTopic.title && currentTopic.frequency) {
         setStepValidation({ ...stepValidation, [currentStep]: true });
       }
     } else if (currentStep === 3 || currentStep === 4) {
       if (
-        !currentTopic.leaderQuestions.length ||
-        !currentTopic.memberQuestions.length
+        !currentTopic.defaultsurvey.questions.filter(q => q.leader).length ||
```

Comment:
Again, could be nicer to move the filter logic up, or handle it in the reducer so we don't have to filter at every step.  This should also improve performance.